Dockerfile

```diff
@@ -1,6 +1,27 @@
-FROM alpine:3.7
-MAINTAINER Nic Cope <n+docker@rk0n.org>
+FROM node as node
+ADD frontend/ .
+RUN npm install && npm run build
+
+FROM golang as golang
```

Comment:
I'd rather pin this to a particular version of the image, say `1.10-alpine-3.7`.

Reply:
Great, I suggest to pin to `1.10-alpine`, it shouldn't be necessary to pin the alpine version. Since the alpine golang image does not have git binary, I'll `apk add` that to have `go get` working.